tests/integration/defs/test_e2e.py

```diff
@@ -1282,19 +1419,26 @@ def test_ptp_quickstart_advanced_deepseek_v3_2nodes_8gpus(
     # "RCCA https://nvbugs/5163844"
     print(f"Testing {model_name}.")
     example_root = Path(os.path.join(llm_root, "examples", "pytorch"))
-    llm_venv.run_cmd([
-        str(example_root / "quickstart_advanced.py"),
-        "--enable_overlap_scheduler",
-        "--model_dir",
-        f"{llm_models_root()}/{model_path}",
-        "--moe_ep_size=8",
-        "--tp_size=16",
-        "--use_cuda_graph",
-        "--kv_cache_fraction=0.5",
-        "--max_batch_size=32",
-        "--max_num_tokens=2048",
-        "--kv_cache_enable_block_reuse",
-    ])
+    running_log = None
+    try:
+        running_log = tempfile.mkstemp(dir="./", suffix=f".{model_name}.log")[1]
+        llm_venv.run_cmd([
+            str(example_root / "quickstart_advanced.py"),
+            "--enable_overlap_scheduler",
+            "--model_dir",
+            f"{llm_models_root()}/{model_path}",
+            "--moe_ep_size=8",
+            "--tp_size=16",
+            "--use_cuda_graph",
+            f"--kv_cache_fraction={_MEM_FRACTION_50}",
+            "--max_batch_size=32",
+            "--max_num_tokens=2048",
+        ],
+                         running_log=running_log)
+        # _check_mem_usage(running_log, [56.30, 0, 0, 0])
+    finally:
+        if running_log:
```

Comment:
A nit.

I think using RAII style https://docs.python.org/zh-cn/3/library/tempfile.html#tempfile.NamedTemporaryFile will automatically delete the files after exception/function returns.

Might make this simpler.